src/ocean/util/DeepReset.d

```diff
@@ -1,12 +1,10 @@
 /*******************************************************************************
 
-    Utility to recursively reset fields of struct to their .init value while
-    preserving array pointers (their length is set to 0 but memory is kept
-    available for further reusage)
+    Utility to recursively reset fields of an aggregate to their .init value
+    while preserving array pointers (their length is set to 0 but memory is kept
+    available for further reuse)
 
-    Copyright:
-        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
-        All rights reserved.
+    Copyright: Copyright (c) 2017 sociomantic labs GmbH. All rights reserved
```

Comment:
I believe you have to keep the original date...?

Reply:
There's only a single year in our "standard" copyright statement present in https://github.com/sociomantic/backend/blob/master/doc/coding/d-style-guide.rst#modules. So should this be set to 2009?

Reply:
`2009-2017` is the solution.

Reply:
>There's only a single year in our "standard" copyright statement

Ah, that's an example for something that's been in copyright since 2017. For things that have been copyrighted for longer, you should specify the year range, as Nemanja says.